templates/_helpers.tpl

```diff
@@ -435,8 +435,8 @@ Compile all warnings into a single message, and call fail.
 {{- $messages = append $messages (include "deploy.validateValues.ingress.tls" .) -}}
 {{- $messages = append $messages (include "deploy.validateValues.keystore.passphrase" .) -}}
 {{- $messages = append $messages (include "deploy.validateValues.license" .) -}}
```

Comment:
validation fails with default values. would be useful to either remove the validation or add non empty default values or appropriate default values for validation to pass.

Reply:
What is the concrete problem here? I didn't notice the problem. Validation is here to check missing configuration keys.

Reply:
The values for these fields is empty in values yaml.